cucumber/authenticators_azure/features/authn_azure_hosts.feature

```diff
@@ -177,3 +177,19 @@ Feature: Azure Authenticator - Different Hosts can authenticate with Azure authe
     """
     Errors::Authentication::Security::RoleNotAuthorizedOnWebservice
     """
+
+  # This test runs a failing authentication request that is already
+  # tested in another scenario (Host without resource-group annotation is denied).
+  # We run it again here to verify that we write a message to the audit log
+  Scenario: Authentication failure is written to the audit log
+    And I have host "no-resource-group-app"
+    And I set subscription-id annotation to host "no-resource-group-app"
+    And I grant group "conjur/authn-azure/prod/apps" to host "no-resource-group-app"
+    And I fetch a non-assigned-identity Azure access token from inside machine
+    And I save my place in the log file
```

Comment:
it audit log here, right? this test looks the same as "Host without resource-group annotation is denied"  beside the last line in logs, but the user need to understand it different log. I would write " And I save my place in the audit_log file".. all over the places related to audit_logs

Reply:
that is correct and i wrote a comment about it just before the scenario.

Reply:
I saw your comment and its great, but writing audit_logs instead of log can help(during reading test results if we have some failures)

Reply:
i changed the code a bit to make it clearer that we look for audit messages (although they are in the main log file)